lib/Config.js

```diff
@@ -654,8 +654,14 @@ class Config extends EventEmitter {
             if (config.vaultd.host !== undefined) {
                 assert.strictEqual(typeof config.vaultd.host, 'string',
                                    'bad config: vaultd host must be a string');
-                this.vaultd.host = config.vaultd.host;
             }
+            if (process.env.VAULTD_HOST !== undefined) {
+                assert.strictEqual(typeof process.env.VAULTDHOST, 'string',
+                    'bad config: vaultd host must be a string');
+            }
```

Comment:
can probably eliminate the need for the ternary, by assigning here
```suggestion
                this.vaultd.host = process.env.VAULTD_HOST;
            }
```